spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -93,27 +93,6 @@ public class DefaultSubscriberFactory implements SubscriberFactory {
 
   private Code[] retryableCodes;
 
-  /**
-   * Default {@link DefaultSubscriberFactory} constructor.
-   *
-   * @param projectIdProvider provides the default GCP project ID for selecting the subscriptions
-   * @deprecated Use the new {@link DefaultSubscriberFactory
-   *     (GcpProjectIdProvider,PubSubConfiguration)} instead
-   */
-  @Deprecated
-  public DefaultSubscriberFactory(GcpProjectIdProvider projectIdProvider) {
```

Comment:
This may need to be documented since this is a public constructor?

Reply:
Resolving the comment since this is already being discussed in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1287#discussion_r1045979188